specification/communitytraining/resource-manager/Microsoft.Community/stable/2023-11-01/CommunityTrainings.json

```diff
@@ -816,6 +816,24 @@
           }
         ]
       }
+    },
```

Comment:
Should we add omit-unreachable-code to avoid changes to the swagger?

Reply:
so the ones I didn't add it is because when we add that flag then it omit other models. So I think its better to have this for now and we'll get to omitting version automatically soon.